src/pages/Swap/AdvancedSwapMode/AdvancedSwapMode.stories.mdx

```diff
@@ -0,0 +1,161 @@
+import { Token } from '@swapr/sdk'
+import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs'
+
+import { Trade } from './Trade'
+import { Chart } from './Chart'
+import { ColumnHeader } from './ColumnHeader'
+import { InfiniteScroll } from './InfiniteScroll'
+
+import { BaseWrapper, TransactionsWrapper } from './AdvancedSwapMode.styles.ts'
+
+<Meta title="AdvancedSwapModeMDX" />
```

Comment:
I don't think it's necessary to have MDX on the title, it's just a detail.
(we also have to remove it from ButtonsMDX.

```suggestion
<Meta title="AdvancedSwapMode" />
```

Reply:
It could help with navigation. It is for discussion as Buttons and CollorPallets contain it.